pkg/kubelet/network/dns/dns.go

```diff
@@ -156,15 +156,15 @@ func (c *Configurer) CheckLimitsForResolvConf() {
 	f, err := os.Open(c.ResolverConfig)
 	if err != nil {
 		c.recorder.Event(c.nodeRef, v1.EventTypeWarning, "CheckLimitsForResolvConf", err.Error())
-		glog.Error("CheckLimitsForResolvConf: " + err.Error())
+		glog.V(4).Infof("CheckLimitsForResolvConf: " + err.Error())
```

Comment:
Why change the log level from Error to Info?

Reply:
TBH, I don't think it is an error. We are anyway creating the pod. It probably would have made sense if we stopped the pod creation/update event based on this error.

Reply:
i agree that since we still start the pod, its not really an error.